edi_oca/models/edi_exchange_record.py

```diff
@@ -332,6 +332,42 @@ def action_exchange_receive(self):
         self.ensure_one()
         return self.backend_id.exchange_receive(self)
 
+    def action_exchange_send_on_max_retries_reached(self):
```

Comment:
These methods should be renamed based on the new implementation on qjob side.
They should be private BTW.